src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -346,6 +350,19 @@ export default class DefaultAudioVideoController
       this._reconnectController.startedConnectionAttempt(true);
     }
 
+    if (!useAudioConnection) {
+      this.meetingSessionContext.transceiverController = new VideoOnlyTransceiverController(
+        this.logger,
+        this.meetingSessionContext.browserBehavior
+      );
+    }
```

Comment:
Move this up and integrate it with line 248.

Reply:
k, i moved them together.

Reply:
I don't see these moved. Try to only assign `transceiverController` once.